builder-classic-22/builder.toml

```diff
@@ -0,0 +1,111 @@
+description = "Classic (shimmed) buildpack builder for Heroku-22 stack, based on ubuntu:22.04 base image"
+
+[stack]
+id = "heroku-22"
+build-image = "heroku/heroku:22-cnb-build"
+run-image = "heroku/heroku:22-cnb"
+
+[lifecycle]
+version = "0.14.0"
+
+[[buildpacks]]
+  id = "heroku/java"
+  uri = "https://cnb-shim.herokuapp.com/v1/heroku/java?version=0.3.1&name=Java"
+
+[[buildpacks]]
+  id = "heroku/go"
+  uri = "https://cnb-shim.herokuapp.com/v1/heroku/go?version=0.3.1&name=Go"
+
+[[buildpacks]]
+  id = "heroku/nodejs"
+  uri = "https://cnb-shim.herokuapp.com/v1/heroku/nodejs?version=0.3.1&name=Node.js"
+
+[[buildpacks]]
+  id = "heroku/php"
+  uri = "https://cnb-shim.herokuapp.com/v1/heroku/php?version=0.3.1&name=PHP"
+
+[[buildpacks]]
+  id = "heroku/procfile"
+  uri = "docker://docker.io/heroku/procfile-cnb:1.0.1"
+
+[[buildpacks]]
+  id = "heroku/python"
+  uri = "https://cnb-shim.herokuapp.com/v1/heroku/python?version=0.3.1&name=Python"
+
+[[buildpacks]]
+  id = "heroku/ruby"
+  uri = "https://cnb-shim.herokuapp.com/v1/heroku/ruby?version=0.3.1&name=Ruby"
+
+[[buildpacks]]
+  id = "heroku/scala"
+  uri = "https://cnb-shim.herokuapp.com/v1/heroku/scala?version=0.3.1&name=Scala"
+
+[[order]]
```

Comment:
Should this order match the default Heroku detection order?

If so, that order is:
`ruby clojure python java gradle scala php go nodejs`

(from https://github.com/heroku/codon/blob/dd40cf73232a39e55279a9f03bdcab9c924ecc32/lib/codon/buildpacks.rb#L9)

Reply:
This order was ported from `heroku/buildpacks:20`, but yeah, I think the platform order makes more sense.